source/intro.tex

```diff
@@ -833,18 +833,17 @@
 \indextext{conformance requirements!library|)}
 
 \pnum
-An implementation is either a
-\defnadj{hosted}{implementation} or a
-\defnadj{freestanding}{implementation}.
-A freestanding
-implementation is one in which execution may take place without the benefit of
-an operating system.
-A hosted implementation
-supports all the facilities described in this document, while
-a freestanding implementation
+A \defnadj{freestanding}{implementation}
+is one in which execution may take place
+without the benefit of an operating system.
+A \defnadj{hosted}{implementation}
+supports all the facilities described in this document,
+while a freestanding implementation
 supports the entire \Cpp{} language
 described in \ref{lex} through \ref{\lastcorechapter} and
 the subset of the library facilities described in \ref{compliance}.
+It is \impldef{whether the implementation is hosted or freestanding}
+whether the implementation is hosted or freestanding.
```

Comment:
Can it be both? Can it be neither? With the old wording, clearly no ("An implementation is either a hosted implementation or a freestanding implementation").